openwisp_firmware_upgrader/tests/test_private_storage.py

```diff
@@ -68,9 +74,41 @@ def test_staff_operator_with_same_organization(self):
             user=staff_user, organization=self.test_org, is_admin=True
         )
         self.client.force_login(staff_user)
-        self._download_firmware_assert_status(200)
+        self._download_firmware_assert_status(403)
 
     def test_superuser(self):
         user = self._get_admin()
         self.client.force_login(user)
         self._download_firmware_assert_status(200)
+
+    def test_view_permission_check(self):
```

Comment:
Are you sure this test is correct? I believe the code comments are outdated. 

A user being **Organization Admin** and a user having **Administrator** group are two different things. Is this clear?

Reply:
Yes clear now